test/compilable/test17752.d

```diff
@@ -0,0 +1,16 @@
+// REQUIRED_ARGS: -c -de
```

Comment:
Generally, please put a link to the bugzilla issue that is addressed.

Reply:
Ok done.